Examples.md

```diff
@@ -21,6 +21,26 @@ int main(void) {
 }
 ```
 
+## C++
+
+```c++
+#include <cstdio>
+#include <cstdlib>
+#include <string>
+#include <iostream>
+
+int writeOut(const std::string& text)
+{
+  std::cout << text;
+  return fflush(stdout);
```

Comment:
Would you mind writing this example in terms of just C++ iostream APIs, instead of mixing in C APIs here? For code using `<iostream>`, it would be more idiomatic to use `<iostream>` for everything.

Reply:
You are right. It can be done in less code. Thank you!